src/data/faq.json

```diff
@@ -35,8 +35,8 @@
-                            "<div class='container'><div class='row'><div class='col-4'><a href='../../assets/img/faq/low_risk_en.png' target='_blank'><figure><img class='img-fluid' src='../../assets/img/faq/low_risk_preview_en.png' alt='image'></a><figcaption aria-hidden='true'><em>Click on the image to get a complete view</em></figcaption></figure></div><div class='col-8'><b>Exposure with low-risk</b><br/>In the case of a low-risk encounter, the risk card remains green because there is no acute need for action on their part. The risk of such an encounter is classified as low due to the duration of the encounter and the distance between the devices. You are still advised to adhere to the general distance and hygiene rules. You can find these by tapping on the risk card. <br/> <br/> <br/>  <b>Exposure with increased risk</b> <br/>In an encounter with increased risk, the risk card turns red. You can find the corresponding action instructions by tapping on the risk card.</div></div>",
-                            "<div class='container'><div class='row'><div class='col-sm'><a href='../../assets/img/faq/high_risk_en.png' target='_blank'><figure><img class='img-fluid' src='../../assets/img/faq/high_risk_preview_en.png' alt='image'></a><figcaption aria-hidden='true'><em>Click on the image to get a complete view</em></figcaption></figure></div><div class='col-sm'><figure><img class='img-fluid' src='../../assets/img/faq/hr_en.png' alt='image'><figcaption aria-hidden='true'><em>The hygiene rules are displayed in the Corona-Warn-App</em></figcaption></figure></div><div class='col-sm'><a href='../../assets/img/faq/ansteckungsrisiko_minimieren_en.png' target='_blank'><figure><img class='img-fluid' src='../../assets/img/faq/ansteckungsrisiko_minimieren_en_preview.png' alt='image'></a><figcaption aria-hidden='true'><em>Click on the image to get a complete view</em></figcaption></figure></div></div></div>",
+                            "<div class='container'><div class='row'><div class='col-4'><a href='../../assets/img/faq/low_risk_en.png' target='_blank'><figure><img class='img-fluid' src='../../assets/img/faq/low_risk_preview_en.png' alt=''></a><figcaption aria-hidden='true'><em>Click on the image to get a complete view</em></figcaption></figure></div><div class='col-8'><b>Exposure with low-risk</b><br/>In the case of a low-risk encounter, the risk card remains green because there is no acute need for action on their part. The risk of such an encounter is classified as low due to the duration of the encounter and the distance between the devices. You are still advised to adhere to the general distance and hygiene rules. You can find these by tapping on the risk card. <br/> <br/> <br/>  <b>Exposure with increased risk</b> <br/>In an encounter with increased risk, the risk card turns red. You can find the corresponding action instructions by tapping on the risk card.</div></div>",

```

Comment:
Not sure about this one. This seems to be an empty link because the link has no text, only an image. It would make sense to add something useful here.